modules/virt-configuring-live-migration-limits.adoc

```diff
@@ -25,18 +25,32 @@ $ oc edit configmap kubevirt-config -n openshift-cnv
 [source,yaml]
 ----
 apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: kubevirt-config
-  namespace: kubevirt
-  labels:
-    kubevirt.io: ""
 data:
-  feature-gates: "LiveMigration"
+  default-network-interface: masquerade
+  feature-gates: DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar,Snapshot
   migrations: |-
     parallelMigrationsPerCluster: "5"
     parallelOutboundMigrationsPerNode: "2"
     bandwidthPerMigration: "64Mi"
     completionTimeoutPerGiB: "800"
     progressTimeout: "150"
+  machine-type: pc-q35-rhel8.3.0
+  selinuxLauncherType: virt_launcher.process
+  smbios: |-
+    Family: Red Hat
+    Product: Container-native virtualization
+    Manufacturer: Red Hat
+    Sku: 2.6.0
+    Version: 2.6.0
+kind: ConfigMap
+metadata:
+  creationTimestamp: "2021-03-26T18:01:04Z"
+  labels:
+    app: kubevirt-hyperconverged
+  name: kubevirt-config
+  namespace: openshift-cnv
+  resourceVersion: "15371295"
+  selfLink: /api/v1/namespaces/openshift-cnv/configmaps/kubevirt-config
+  uid: <uuid>
+
```

Comment:
Looks like it doesn't show up in the rendered version, but did notice an extra blank line here in source.

Reply:
Removed.